data/net/minecraft/server/commands/ScheduleCommand.mapping

```diff
@@ -1,4 +1,23 @@
 CLASS net/minecraft/server/commands/ScheduleCommand
+	METHOD lambda$register$2 (Lnet/minecraft/commands/CommandSourceStack;)Z
+		ARG 0 source
+	METHOD lambda$register$3 (Lcom/mojang/brigadier/context/CommandContext;)I
+		ARG 0 context
+	METHOD lambda$register$4 (Lcom/mojang/brigadier/context/CommandContext;)I
+		ARG 0 context
+	METHOD lambda$register$5 (Lcom/mojang/brigadier/context/CommandContext;)I
+		ARG 0 context
+	METHOD lambda$register$6 (Lcom/mojang/brigadier/context/CommandContext;)I
+		ARG 0 context
+	METHOD lambda$schedule$10 (Lnet/minecraft/resources/ResourceLocation;ZLnet/minecraft/world/level/timers/TimerQueue;JLnet/minecraft/commands/CommandSourceStack;ILjava/util/Collection;)V
+		ARG 7 functionCollection
```

Comment:
```suggestion
		ARG 7 functions
```

imo `functionCollection` is ungangly, this describes what it is better (either a single function, or functions plural). The type in question is the right of a `Either<CommandFunction, Collection<CommandFunction>>>`